spec/spec_helper.rb

```diff
@@ -1,31 +1,70 @@
+ENV["RAILS_ENV"] ||= 'test'
+ENV["AUTOMATE_DOMAINS"] = "ManageIQ" # Reset only the ManageIQ automate domain when testing.
+
+require File.expand_path("../manageiq/config/environment", __FILE__)
+require 'rspec/rails'
 require 'simplecov'
-SimpleCov.start
 
-require 'manageiq-content'
+SimpleCov.start
 
-# Reset only the ManageIQ automate domain when testing.
-ENV["AUTOMATE_DOMAINS"] = "ManageIQ"
+if ENV["TRAVIS"] || ENV['CI']
+  require 'coveralls'
+  Coveralls.wear!('rails') { add_filter("/spec/") }
+end
 
 def require_domain_file
-  spec_name = caller_locations.first.path
+  spec_name = caller_locations(1..1).first.path
```

Comment:
It looks like this method can use the `domain_file` method below and reduce the duplication